vita3k/modules/SceDriverUser/SceMotion.cpp

```diff
@@ -48,28 +51,76 @@ EXPORT(int, sceMotionGetDeadbandExt) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceMotionGetDeviceLocation) {
-    TRACY_FUNC(sceMotionGetDeviceLocation);
+EXPORT(int, sceMotionGetDeviceLocation, SceMotionDeviceLocation *devLocation) {
+    TRACY_FUNC(sceMotionGetDeviceLocation, devLocation);
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceMotionGetGyroBiasCorrection) {
+EXPORT(SceBool, sceMotionGetGyroBiasCorrection) {
     TRACY_FUNC(sceMotionGetGyroBiasCorrection);
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceMotionGetMagnetometerState) {
+EXPORT(SceBool, sceMotionGetMagnetometerState) {
     TRACY_FUNC(sceMotionGetMagnetometerState);
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceMotionGetSensorState) {
-    TRACY_FUNC(sceMotionGetSensorState);
+EXPORT(int, sceMotionGetSensorState, SceMotionSensorState *sensorState, int numRecords) {
+    TRACY_FUNC(sceMotionGetSensorState, sensorState, numRecords);
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceMotionGetState) {
-    TRACY_FUNC(sceMotionGetState);
+EXPORT(int, sceMotionGetState, SceMotionState *motionState) {
+    TRACY_FUNC(sceMotionGetState, motionState);
+    if (!motionState)
+        return RET_ERROR(SCE_MOTION_ERROR_NULL_PARAMETER);
+
+    STUBBED("Set some default values");
```

Comment:
```suggestion
    STUBBED("Set to zero");
```

Reply:
i no set all to zero, is more default value in default position